data/pug.txt

```diff
@@ -136,12 +136,13 @@ effect "pug anti-missile"
 outfit "Tier 3 Anti-Missile"
 	category "Turrets"
 	cost 12000000
-	thumbnail "outfit/unknown"
+	thumbnail "outfit/T3 anti missile"
```

Comment:
I'm a bit uncomfortable about how the T is capitalized here.

Reply:
Corrected